SpatialGDK/Source/SpatialGDK/Public/Schema/UnrealMetadata.h

```diff
@@ -85,7 +85,9 @@ struct UnrealMetadata : Component
 		{
 			Class = FindObject<UClass>(nullptr, *ClassPath, false);
 		}
-		else
+
+		// It's possible we were unable to load a stably named actor as all references have been deleted.
+		if (!Class)
```

Comment:
Just so we're on the same page - does that mean that we believe the object is stably named, but unreal doesn't? If Unreal did, we wouldn't be deleting it when it goes out of scope.

Reply:
True, we have a `StablyNamedRef` and `Object->IsFullNameStableForNetworking()` is false.

Reply:
Stably named refs are given to these placed actors due to this PR: https://github.com/spatialos/UnrealGDK/pull/721
```
	// We want to have a stably named ref if this is a loaded Actor.
	// We use this to indicate if a new Actor should be created or to link a pre-existing Actor when receiving an AddEntityOp.
	// Previously, IsFullNameStableForNetworking was used but this was only true if bNetLoadOnClient was true.
```
Looks like it's to help with server re-connection.